sdk/simulation/urtssim/enclave_creator_sim.cpp

```diff
@@ -118,30 +108,22 @@ int EnclaveCreatorSim::create_enclave(secs_t *secs, sgx_enclave_id_t *enclave_id
         LockGuard lock(&s_enclave_info_mutex);
         if (s_enclave_elrange_map.count(base_address) != 0)
         {
-            enclave_elrange_t *enclave_elrange = s_enclave_elrange_map[base_address];
-            if (enclave_elrange == NULL)
-            {
-                return SGX_ERROR_UNEXPECTED;
-            }
+            enclave_elrange_t enclave_elrange = s_enclave_elrange_map[base_address];
```

Comment:
This is a copy that will be thrown out. Do you want a mutable reference to s_enclave_elrange_map[base_address]?